src/libpriv/rpmostree-core.c

```diff
@@ -223,55 +223,6 @@ rpmostree_treespec_to_variant (RpmOstreeTreespec *spec)
 }
 
 /***********************************************************
- *                    RpmOstreeInstall                     *
- ***********************************************************
- */
-
-struct _RpmOstreeInstall {
-  GObject parent;
-
-  GPtrArray *packages_requested;
-
-  /* Target state -- these are populated during prepare_install() */
-  GPtrArray *packages_to_download;
-  GPtrArray *packages_to_import;
-  GPtrArray *packages_to_relabel;
-
-  guint64 n_bytes_to_fetch;
```

Comment:
I think the idea I had in my head at the time I introduced the `RpmOstreeInstall` was to expose this metric, then a higher level tool could render it.  But in practice right now we're just rendering text in the core.  Which would be hard to change.  So this makes sense to me!